### DISCUSSION TITLE
Welcome to apash Discussions!

### DISCUSSION THREAD

**Discussion by hastec-fr** [hastec-fr/apash#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aurelien-mariage** [comment]
Congratulations for this wonderful initiative !! I installed it and will let you know next time I'll have to write scripts how it was.
